frontend/src/k8sTypes.ts

```diff
@@ -598,7 +598,7 @@ export type PipelineRunTaskSpec = {
   stepTemplate?: {
     volumeMounts?: PipelineRunTaskVolumeMount[];
   };
-  results: PipelineRunTaskSpecResult[];
+  results: PipelineRunTaskSpecResult[] | undefined;
```

Comment:
Need to compare against tekton spec.

Reply:
https://tekton.dev/docs/pipelines/pipeline-api/#tekton.dev/v1.TaskSpec indicates that `results` is not optional